modules/configuring-private-storage-endpoint-azure-user-provided-vnet-subnet.adoc

```diff
@@ -0,0 +1,102 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/configuring-private-cluster.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-private-storage-endpoint-azure-user-provided-vnet-subnet"]
+= Configuring a private storage endpoint on Azure with user-provided VNet and subnet names
+
+Use the following procedure to configure a private storage endpoint on Azure with user-provided VNet and subnet names.
+
+.Prerequisites 
+
+* You have configured the Image Registry Operator for use with Azure.
```

Comment:
What does this mean exactly? Is it about having the `ImageRegistry` capability enabled?

Reply:
I guess I'm not really sure. What are the required prerequisites beyond "You must know the VNet and subnet names used in your Azure environment." ? 

I assume that they should have configured a private Azure cluster, right?

Reply:
With a public cluster users will still be able to use this feature (although it doesn't make as much sense).
I guess the only prerequisite is that there is a registry in their cluster? And that the storage is managed by the operator (`.spec.storage.managementState: Managed`) but but these two things are defaults on new installations so I'm not sure if it's worth mentioning 🤔

Reply:
Added the prerequisite: * You have configured an image registry to run on Azure